### ISSUE TITLE
Add anything and everything to a popup

### ISSUE THREAD

**Issue by johnpauls** [rstudio/leaflet#120]
It would be nice to be able to add anything to a popup - plots, tables etc. This would be particularly good for shiny applications where the various render functions could be used to make graphs or tables that update based on both user selections at the level of the map and within the popup itself.

**Comment by ziyadsaeed** [comment]
something as basic as
```R
popup = ~as.character(tags$b("City"))
```
doesn't work, you have to use paste or sprintf

**Comment by efh0888** [comment]
I completely agree. In my app I was hoping to include a nice DataTable in popups.
